bin/main.rs

```diff
@@ -173,14 +174,23 @@ async fn main() -> anyhow::Result<()> {
     .await?;
 
     // TODO: We will probably need to set up all the flags (like lattice prefix and topic prefix) down the line
-    let connection_pool = ControlClientConstructor::new(
+    let connection_pool: ControlClientConstructor<CachedKvStore> = ControlClientConstructor::new(
         client.clone(),
         ControlClientConfig {
-            js_domain: args.domain,
+            js_domain: args.domain.clone(),
             topic_prefix: None,
         },
     );
 
+    let hack_connection_pool: ControlClientConstructor<DirectKvStore> =
+        ControlClientConstructor::new(
```

Comment:
I think this is leftover from testing

Reply:
Actually it is meant to be there, but I didn't change the name from testing. Will change now